### DISCUSSION TITLE
Could you please share the new G4E test version from today?

### DISCUSSION THREAD

**Discussion by Taviruni** [chenall/grub4dos#331] [General]
Could you share the new test version?

From this [post](http://bbs.c3.wuyou.net/forum.php?mod=redirect&goto=findpost&ptid=422652&pid=4371535) of yaya.

I'm not able to download from  that forum, and I would like to test it.

Thanks in advance.

alacran (from reboot.pro

**Comment by a1ive** [comment]
[a.rar](https://github.com/chenall/grub4dos/files/7115129/a.rar.txt)

**Comment by steve6375** [comment]
How does chainloader find NTBOOT? I do not use /efi/grub/ext folder.

**Reply by Taviruni** [reply]
Using the test version, and after copy NTBOOT to /efi/grub/ext/

Having /WinPE/Win10XPE_x64.wim into (hd1,0) FAT-32 first primary active partition of second internal HD (MBR formatted)

Following commands:

title WinPE/Win10XPE_x64.wim 765 MB UEFI Boot by NTBOOT -0
chainloader /boot/WinPE/Win10XPE_x64.wim

Din't work, please see attached picture:

![06154611](https://user-images.githubusercontent.com/35506636/132263361-3849dbbd-3d34-4fab-ba68-2e787aea4e5f.png)

But if using ntloader with following commands:

title Boot WinPE/Win10XPE_x64.wim using ntloader
set nt=ntloader
find /WinPE/Win10XPE_x64.wim | set ptn=
uuid %ptn% > nul ;; set UUID=%?%
kernel /%nt% hires=0 uuid=%UUID% file=/WinPE/Win10XPE_x64.wim
initrd /initrd.lz1

It boots very fine, from that partition or any other primary partition on that disk.

So it seems something is wrong in implementing NTBOOT on G4E.

alacran (from reboot.pro)

**Comment by steve6375** [comment]
same issue with error 15 file not found on an NTFS partition (ntboot is loaded OK as 0x82).
Also, If wim file is on (hd0,1) FAT32 partition then chainloader  (hd0,1)/boot.wim command hangs!

For location of NTBOOT, prefer to specify path as parameter on chainloader command line

chainloader [--ntboot=(path to ntboot)] FILE | DEVICE

if no --ntboot then use` %@BOOT%/EFI/GRUB/EXT/NTBOOT ` else use path specified by --ntboot

chainload --ntboot=(hd0,0)/myext/ntboot (hd0,0)/boot.wim

It also allows for easy testing for different ntboot files.

**Comment by steve6375** [comment]
Cannot download latest BOOTX64.rar from http://bbs.c3.wuyou.net/
Can someone add it here please?
